packages/react/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "src/**/*.js",
     "src/**/*.ts",
     "src/**/*.tsx",
+    "!src/**/*.figma.tsx",
```

Comment:
Let me know if I'm misunderstanding, it seems like we would want this in our `type-check` task for CI to make sure that figma connect files are generated correctly but we would not want it in our package files, is that correct?

Reply:
I think neither.

- def. not in package file
- there are some issues with making it typescript compliant as it is "read as text", this is why I should probably also not be checked.

Happy to jump on a pairing session to see if there is some way around ignoring it.

Reply:
@lukasoppermann sounds great, would love to pair up on it if you have the time!